Dockerfile.mysql

```diff
@@ -2,13 +2,8 @@
 FROM mysql:latest
 
 # Environment variables for MySQL configuration
-ENV MYSQL_ROOT_PASSWORD=1234
-ENV MYSQL_DATABASE=app_db
-ENV MYSQL_USER=app_user
-ENV MYSQL_PASSWORD=1234
 
 # Expose port 3306 for MySQL connections
-EXPOSE 3306
+
```

Comment:
The `EXPOSE 3306` command is commented out. This is necessary to allow external connections to the MySQL database. Please uncomment this line.